src/main/kotlin/blackjack/domain/player/Player.kt

```diff
@@ -1,13 +1,17 @@
 package blackjack.domain.player
 
+import blackjack.domain.card.CardScore
 import blackjack.domain.card.Cards
-import blackjack.domain.score.CardScore
+import blackjack.domain.common.Money
 
 class Player(
     name: String,
     cards: Cards = Cards.empty(),
-    private var playerStatus: PlayerStatus = PlayerStatus.HIT
+    playerStatus: PlayerStatus = PlayerStatus.HIT,
+    private val batting: Money = Money.ZERO,
 ) : Participant(name, cards) {
+    private var _playerStatus: PlayerStatus = playerStatus
+    val playerStatus get() = _playerStatus
```

Comment:
그 전에 제가 예시로 드린 부분에 대해서 클라이언트가 이름을 지정할 때 번거로움이 있지 않을까해요.

`Player(_playerStatus =  PlayerStatus.HIT)` 처럼 _ 를 하나 더 쓰게 되는거죠. 그래서 더 좋고 간결하게 갈 수 있는 방법이 없나 생각을 했는데, 

```suggestion
        playerStatus: PlayerStatus = PlayerStatus.HIT,
) : Participant(name, cards) {
    var playerStatus: PlayerStatus = playerStatus
        private set
```

위의 예제처럼 set에 대해서는 접근제어를 private로 주는 방법이 더 좋겠다는 생각이 드네요! 😊